### ISSUE TITLE
Update to use JS tracking

### ISSUE THREAD

**Issue by RCheesley** [mautic/mautic-joomla#24]
Will the plugin be updated to support the new JS tracking? Should we disable it until it is to avoid loading image + JS or can we not use JS until this is updated (plugin is being used to render forms)

**Comment by escopecz** [comment]
You can still use the tracking pixel unless you implement the JS tracking yourself. In that case the page hits would double. But you're right that we should switch to JS tracking and prepare for Mautic 2.0.

**Comment by wderuijter** [comment]
Indeed, we are unable to switch our Joomla sites to using JS tracking with it's speed benifits.

**Comment by wderuijter** [comment]
Any updates on this?

**Comment by escopecz** [comment]
It seems to be already implemented in the latest code on GitHub (https://github.com/mautic/mautic-joomla/blob/master/mautic.php#L106) Try to download the latest code and install/update the plugin and let us know.

https://github.com/mautic/mautic-joomla/archive/master.zip

**Comment by wderuijter** [comment]
Hey @escopecz , I've just updated the plugin. Now I see in chrome dev tools 2 requests to mtc.js (1 script and 1 xhr) and a single request to mtracking.gif.

I can't see mtracking.gif in the page source. Am I correct to assume the request to the tracking pixel is done through JS?

**Comment by escopecz** [comment]
That's not good. Can you provide your website URL so I could look at the source?

**Comment by escopecz** [comment]
I looked to the plugin source again and there is only the JS tracking now. The tracking pixel was removed, so that should make only one tracking request per page load.

**Comment by wderuijter** [comment]
Hi @escopecz , 

the website in this case for instance https://www.soetkees.nl/bestemmingen/europa/607-belgrado
Opening the site incognito only loads the mtc.js once and the mtracking.gif once.
I'm running mautic 2.0.1 and installed the joomla plugin in the archive you mentioned earlier.

**Comment by wderuijter** [comment]
Thanks for clearing this up!
